src/hash/rescue/rpo/digest.rs

```diff
@@ -242,7 +242,12 @@ impl TryFrom<[u64; DIGEST_SIZE]> for RpoDigest {
             return Err(RpoDigestError::InvalidInteger);
         }
 
-        Ok(Self([value[0].into(), value[1].into(), value[2].into(), value[3].into()]))
+        Ok(Self([
+            value[0].try_into().unwrap(),
+            value[1].try_into().unwrap(),
+            value[2].try_into().unwrap(),
+            value[3].try_into().unwrap(),
+        ]))
     }
```

Comment:
We can probably combine this with the checks above - e.g., maybe using `value[0].try_into().map_err(|_| RpoDigestError::InvalidInteger)?`.